services/dask-sidecar/docker/boot.sh

```diff
@@ -71,7 +71,8 @@ else
   num_gpus=$(python -c "from simcore_service_dask_sidecar.utils import num_available_gpus; print(num_available_gpus());")
 
   # RAM (is computed similarly as the default dask-sidecar computation)
-  ram=$(($(python -c "import psutil; print(int(psutil.virtual_memory().total * $num_cpus/$(nproc)))") - ${DASK_SIDECAR_NON_USABLE_RAM:-0}))
+  _value=$(python -c "import psutil; print(int(psutil.virtual_memory().total * $num_cpus/$(nproc)))")
+  ram=$((_value - ${DASK_SIDECAR_NON_USABLE_RAM:-0}))
```

Comment:
a bit picky here...

Reply:
Sorry, the code hightligh in vscode could not render further this point and this is the only way I found to overcome this.